### PR TITLE
HDDS-7243. Remove RequestFeatureValidator from echoRPC method which supports only ValidationCondition.OLDER_CLIENT_REQUESTS

### PR DESCRIPTION
**Pull request by DaveTeng0**
## What changes were proposed in this pull request?

1. Remove RequestFeatureValidator from echoRPC method in OzoneManagerRequestHandler.java which previously supports only for ValidationCondition.OLDER_CLIENT_REQUESTS
2. Remove EchoRPC command from ratis operation (since the EchoRPC command currently isn't designed to go though ratis. But maybe we'll improve that feature later in order to test performance of Ozone RPC communication in ratis as well.) 

## What is the link to the Apache JIRA
https://issues.apache.org/jira/browse/HDDS-7243

## How was this patch tested?
Current robot test in [hadoop-ozone/dist/src/main/smoketest/freon/echoRPCLoad.robot](https://github.com/apache/ozone/blob/master/hadoop-ozone/dist/src/main/smoketest/freon/echoRPCLoad.robot)

### CODE REVIEW

**Comment by DaveTeng0** [comment]
cc. @kerneltime  @errose28

**Review by kerneltime — APPROVED** [review]

File: hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/utils/OzoneManagerRatisUtils.java [hunk 1]
```diff
@@ -290,8 +289,6 @@ public static OMClientRequest createClientRequest(OMRequest omRequest,
       volumeName = keyArgs.getVolumeName();
       bucketName = keyArgs.getBucketName();
       break;
-    case EchoRPC:
```
Comment (errose28) [inline hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/utils/OzoneManagerRatisUtils.java:293]: While we don't need this here we should probably add it to `OmUtils#isReadOnly`.

File: hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/utils/OzoneManagerRatisUtils.java [hunk 2]
```diff
@@ -290,8 +289,6 @@ public static OMClientRequest createClientRequest(OMRequest omRequest,
...
-      return new OMEchoRPCRequest(omRequest);
```
Comment (errose28) [inline hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/utils/OzoneManagerRatisUtils.java:294]: We can delete the `OmEchoRPCRequest` class

File: hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/utils/OzoneManagerRatisUtils.java (hunk 1 above)
Comment (DaveTeng0) [reply hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/utils/OzoneManagerRatisUtils.java:293]: Yes sure! it's in OmUtils#isReadOnly already! (https://github.com/apache/ozone/blob/master/hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/OmUtils.java#L273)

File: hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/utils/OzoneManagerRatisUtils.java (hunk 2 above)
Comment (DaveTeng0) [reply hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/utils/OzoneManagerRatisUtils.java:294]: Oh sorry i forgot to remove that class! Yes, updated!

Comment (DaveTeng0) [reply hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/utils/OzoneManagerRatisUtils.java:294]: Thanks ethan for the remind!

File: hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/utils/OzoneManagerRatisUtils.java (hunk 1 above)
Comment (errose28) [reply hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/utils/OzoneManagerRatisUtils.java:293]: Oh nice my bad I had an old version of master checked out locally.

**Review by errose28 — APPROVED** [review]
Thanks @DaveTeng0. Just retriggered an unrelated test failure. We can merge this once it is green.

**Pull request merged**